kernel/build-kernel.sh

```diff
@@ -298,22 +312,26 @@ setup_kernel() {
 
 	if [ -d "$kernel_path" ]; then
 		info "${kernel_path} already exist"
-		return
-	fi
-
-	info "kernel path does not exist, will download kernel"
-	download_kernel="true"
-	[ -n "$kernel_version" ] || die "failed to get kernel version: Kernel version is emtpy"
+		if [[ "${force_setup_generate_config}" != "true" ]];then
```

Comment:
nit: I would have added the 'force build' and 'bash debug' bits as separate commits in this PR, just to make it clearer for the reviewers ;-)

Reply:
OK